packages/terra-search-field/src/SearchField.jsx

```diff
@@ -284,34 +297,44 @@ class SearchField extends React.Component {
       : undefined;
 
     return (
-      <div {...customProps} className={searchFieldClassNames}>
-        <div className={cx('input-group')}>
-          <input
-            {...additionalInputAttributes}
-            className={inputClass}
-            type="search"
-            placeholder={placeholder}
-            onChange={this.handleTextChange}
-            disabled={isDisabled}
-            aria-label={inputText}
-            aria-disabled={isDisabled}
-            onKeyDown={this.handleKeyDown}
-            onInput={this.handleInput}
-            ref={this.setInputRef}
-          />
-          {clearButton}
+      <div className={cx('search-container')}>
+        {isLabelVisible
+        && (
+        <label className={cx('label')}>{groupName}</label>
+        )}
+        <div className={cx('search-container')} role="search" aria-label="Search">
+          <div className={cx('search-container')} role="group" aria-label={groupName}>
```

Comment:
If we already have a label.  Can't we use aria-labelledby?  Do we technically need another aria-label if the label will be read by the screen reader?  If the label is visible, is the value read twice?

Reply:
Verified with Eric that having aria-label at these levels is expected. Looks like aria-label being read twice in case label is visible is not a problem.